spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -418,15 +411,178 @@ as part of an individual deployment request prefixed by the `app.<name of applic
 will deploy the time source with 2048MB of memory, while the log sink will use the default 1024MB.
 
 === Using Spring Cloud Config Server
-If using Spring Cloud Config Server as a Cloud Foundry service, the easiest way to externalize the above configuration
-and consume it from the Data Flow server is to use the `spring-cloud-services-starter-config-client` dependency, which
-is included in the standard distribution of the Spring Cloud Data Flow server for Cloud Foundry.
+Spring Cloud Config Server can be used to centralize configuration properties for Spring Boot applications. Likewise,
+both Spring Cloud Data Flow and the applications orchestrated using Spring Cloud Data Flow can be integrated with
+config-server. Let's review the integration steps for both the variants.
+
+==== Spring Cloud Data Flow and Spring Cloud Config Server
+
+* Download 1.4.x release of Spring Boot from http://start.spring.io
+* Add `spring-cloud-dataflow-server-cloudfoundry-autoconfig` dependency pointing to Spring Cloud Data Flow's Cloud Foundry link:https://github.com/spring-cloud/spring-cloud-dataflow-server-cloudfoundry/releases[release]
+* Add `@EnableDataFlowServer` to the downloaded application
+
+[source,java]
+----
+@SpringBootApplication
+@EnableDataFlowServer
+public class DataFlowServer {
+
+	public static void main(String[] args) {
+		new SpringApplication(DataFlowServer.class).run(args);
+	}
+}
+----
+
+* Add  `spring-cloud-services-starter-config-client` dependency in `pom.xml`. A maven example follows.
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-dataflow-server-cloudfoundry-autoconfig</artifactId>
+	<version>VERSION</version>
```

Comment:
The 2nd bullet above points to the cf-server releases tab; I was hoping it might just be enough, but I see your point about someone copying it as-is. I will add a note to describe what `VERSION` could be.